homeassistant/components/fastdotcom/__init__.py

```diff
@@ -51,13 +52,30 @@ async def _request_refresh(event: Event) -> None:
         """Request a refresh."""
         await coordinator.async_request_refresh()
 
+    async def _request_refresh_service(call: ServiceCall) -> None:
+        """Request a refresh via the service."""
+        await coordinator.async_request_refresh()
+
     if hass.state == CoreState.running:
         await coordinator.async_config_entry_first_refresh()
     else:
         # Don't start the speedtest when HA is starting up
         hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STARTED, _request_refresh)
 
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
+    hass.services.async_register(DOMAIN, "speedtest", _request_refresh_service)
+
+    ir.async_create_issue(
```

Comment:
Move the creation of the issue to the service call so only people who actually call the service get this notice